src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -311,6 +313,48 @@ public async Task HandleGetObjectsFromTSqlModelRequest(GetObjectsFromTSqlModelPa
             return;
         }
 
+        /// <summary>
+        /// Handles request to save a publish profile
+        /// </summary>
+        /// <returns></returns>
+        public async Task HandleSavePublishProfileRequest(SavePublishProfileParams parameters, RequestContext<ResultStatus> requestContext)
+        {
+            try
```

Comment:
Maybe I should break my utility methods out into a shared location, 'cause they'd help save us some of this try-catch boilerplate...

Reply:
That will be great, I can consume them if you already have the changes- else can do that in the next PR that I will be working for, for publish options.

Reply:
Next PR is good!

Reply:
Address it in this PR itself !